docs/resources/azure_virtual_wan.md

```diff
@@ -0,0 +1,96 @@
+---
+title: About the azure_virtual_wan Resource
+platform: azure
+---
+
+## azure_virtual_wan
+
+Use the `azure_virtual_wan` InSpec audit resource to test properties related to a Azure Virtual WAN in a given resource group.
+
+## Azure REST API version, Endpoint, and HTTP Client Parameters
+
+This resource interacts with API versions supported by the resource provider. You can define the `api_version` as a resource parameter. If not provided, the latest version is used. For more information, refer to [`azure_generic_resource`](azure_generic_resource.md).
+
+Unless defined, `azure_cloud` global endpoint and default values for the HTTP client is used. For more information, refer to the resource pack [README](../../README.md).
+
+## Availability
+
+### Installation
+
+This resource is available in the [InSpec Azure resource pack](https://github.com/inspec/inspec-azure). For an example, `inspec.yml` file and how to set up your Azure credentials, refer to resource pack [README](../../README.md#Service-Principal).
+
+## Syntax
+
+`name` is a required parameter.
```

Comment:
@sathish-progress Is `resource_group` also a required parameter?

Reply:
yeah, I will add them ..